dashboard/lti_new.py

```diff
@@ -34,6 +35,7 @@
 TA = 'http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistant'
 COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'
 DUMMY_CACHE = 'DummyCache'
+CANVAS_DATA_ID_INCREMENT = settings.CANVAS_DATA_ID_INCREMENT
```

Comment:
Nit-pick: You could just reference `settings.CANVAS_DATA_ID_INCREMENT` if it's not used elsewhere.